examples/ruby/sinatra/gift_giver/lib/api.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require 'sinatra/json'
+require 'httparty'
+
+class Api < Sinatra::Base
+  before { content_type :json }
+
+  get '/attendees' do
+    json Attendee.all
+  end
+
+  get '/attendees/awarded' do
+    json Attendee.awarded.all
+  end
+
+  post '/attendees/draw' do
+    attendee = Attendee.random
```

Comment:
`/attendees/draw` must filter by `where(awarded: false)`

Reply:
It's doing that by using the `.random` method. But yeah, `Attendee.non_infected.random` would be more evident here.